adapters/quick/src/main/scala/caliban/QuickAdapter.scala

```diff
@@ -80,7 +82,11 @@ final class QuickAdapter[R] private (requestHandler: QuickRequestHandler[R]) {
   )(implicit trace: Trace, tag: Tag[R]): RIO[R, Nothing] =
     Server
       .serve[R](routes(apiPath, graphiqlPath = graphiqlPath, uploadPath = uploadPath, webSocketPath = webSocketPath))
-      .provideSomeLayer[R](Server.defaultWithPort(port))
+      .provideSomeLayer[R](
+        ZLayer.succeed(Server.Config.default.port(port))
+          ++ ZLayer.succeed(NettyConfig.default.leakDetection(LeakDetectionLevel.DISABLED))
+          >+> Server.customized
+      )
```

Comment:
PS: I wanted to use `provideSome` but I got slapped with a different variant of [this issue](https://github.com/zio/zio/issues/8936), due to an ambiguity in the implicit `trace`